sdk/src/loom2d/Display/DisplayObject.ls

```diff
@@ -180,7 +180,7 @@ package loom2d.display
             parentCached = value;
         }
 
-        /** If depth sorting is enabled on parent, this will be used to establish draw order. */
+        /** If depth sorting is enabled on parent, this will be used to establish draw order. Higher values are drawn closer. */
```

Comment:
Pretty basic change, but that is some critical information. I am, however, left with some questions:

- What happens if two Display objects have the same depth value? How is that resolved?
- Is there a way for a user to easily get the depth of objects, and reorder them to fit their needs? (For instance, take a display object and put it before or after all other display objects attached to the parent)
- Is there any particular reason this value is a "float" rather than an "int"?
- Higher values being drawn closer breaks the left hand rule. Essentially, the depth represents the "Z" direction, which should increase in value as it moves away from the "camera". This is kind of a moot point, since Loom already breaks the left hand rule with it "Y" direction... but it's something to consider.